pages/docs/tutorials/validate.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Validate AsyncAPI Files"
+description: In this tutorial we'll learn how to validate AsyncAPI files.
+weight: 120
+---
+
+# Introduction
+We'll discuss what it means to validate AsyncAPI files in this tutorial. AsyncAPI files can be validated in two different ways; by validating the AsyncAPI document and by validating messages or events in runtime. 
+
+This tutorial's main focus will be on how AsyncAPI document validation is done.
+
+# Background context
```

Comment:
this tutorial shows basically as the last item on the list of tutorials. I'm not sure if we need to provide this background. It repeats what already other documents should explain, like for example the getting started guide about openapi

Reply:
Oh, should I rewrite a different background context or remove it completely?

Reply:
`Validation of AysncAPI Document` section is giving a good context IMHO, you explain that there are 2 different AsyncAPI validations, 2 scopes of it, 

also intro is also good

IMHO this is enough